vendor/assets/javascripts/components/input.js.jsx

```diff
@@ -35,7 +37,22 @@ var EditoriumInput = React.createClass({
     });
   },
 
+  sectionLength(data) {
+    data = data || this.props.value;
+    return data && data.length > 0 ? JSON.parse(data)['sections'][1].length : 0;
+  },
+
   render() {
+    switch(this.sectionLength()) {
+    case 0:
+        caption = 'Create';
```

Comment:
Add `let caption;` before the switch statement to avoid creating a global variable